newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js

```diff
@@ -213,3 +225,35 @@ export const onAutoSaveProject = (
 
 export const getWriteErrorMessage = (error: Error): MessageDescriptor =>
   t`An error occurred when saving the project. Please try again by choosing another location.`;
+
+export const onRenderNewProjectSaveAsLocationChooser = ({
+  saveAsLocation,
+  setSaveAsLocation,
+}: {
+  saveAsLocation: ?SaveAsLocation,
+  setSaveAsLocation: (?SaveAsLocation) => void,
+}) => {
+  const outputPath = saveAsLocation
+    ? path.dirname(saveAsLocation.fileIdentifier)
+    : app
+    ? findEmptyPathInDefaultFolder(app)
+    : '';
+  if (!saveAsLocation) {
+    setSaveAsLocation({
+      fileIdentifier: path.join(outputPath, 'game.json'),
+    });
+  }
+
+  return (
+    <LocalFolderPicker
+      fullWidth
+      value={outputPath}
+      onChange={newOutputPath =>
+        setSaveAsLocation({
+          fileIdentifier: path.join(newOutputPath, 'game.json'),
```

Comment:
or here

Reply:
(solving - keeping the existing behavior see other conversation)